ngx_pq_module.c

```diff
@@ -849,6 +849,7 @@ static void ngx_pq_event_handler(ngx_http_request_t *r, ngx_http_upstream_t *u)
             goto ret;
         case CONNECTION_SETENV: ngx_log_debug0(NGX_LOG_DEBUG_HTTP, r->connection->log, 0, "CONNECTION_SETENV"); break;
         case CONNECTION_SSL_STARTUP: ngx_log_debug0(NGX_LOG_DEBUG_HTTP, r->connection->log, 0, "CONNECTION_SSL_STARTUP"); break;
+        case CONNECTION_ALLOCATED: ngx_log_debug0(NGX_LOG_DEBUG_HTTP, r->connection->log, 0, "CONNECTION_ALLOCATED"); break;
```

Comment:
This is not acceptable:
1) options must be sorted alphabetically
2) this option appeared only in version 17, so it will not compile with other versions

Reply:
we need something [like](https://github.com/RekGRpth/pg_task/commit/2f814145b9902f8600b8d1beac7ab0a68718eb98#diff-1fe3026a8b097a159b4e2e12096a550c412720c0a516957f29d81e3803f57cfaL41-L46)